src/data/repositories/IncidentActionD2Repository.ts

```diff
@@ -242,10 +247,11 @@ export class IncidentActionD2Repository implements IncidentActionRepository {
             });
     }
 
-    private getProgramStageByFormType(formType: string) {
+    private getProgramStageByFormType(formType: FormType): Id {
         switch (formType) {
             case "incident-action-plan":
                 return RTSL_ZEBRA_INCIDENT_ACTION_PLAN_PROGRAM_STAGE_ID;
+            case "incident-response-actions":
             case "incident-response-action":
```

Comment:
this is quite confusing, change "incident-response-action" to "single-incident-response-action" . 

Make the change in the next PR for v2, so that we are not blocking release for this code review comment.